src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static final SessionFactory instance = initSessionFactory();
+
+    private HibernateUtil() {
+    }
```

Comment:
do we need this private constructor?
initialization of SessionFactory we do in initSessionFactory.